pyproject.toml

```diff
@@ -41,6 +41,7 @@ sphinxcontrib-mermaid = "^0.9.2"
 sphinx-autodoc-typehints = "^2.1.0"
 sphinx-book-theme = "^1.1.2"
 ipywidgets = "^8.1.3"
+jupyter = "^1.0.0"
```

Comment:
Why is this added now?

Reply:
Can't render tutorials without it, at least on my machine. Possible consequence of #388?

Reply:
Tutorials are still being rendered in CI without this change, though?

Reply:
Perhaps installed by the `quarto` installation action? Not sure. But in any case I would argue it should be an explicit docs dep since quarto uses it as the execution engine for the tutorials.